tests/spectron/spectronWindowsActions.js

```diff
@@ -0,0 +1,80 @@
+const robot = require('robotjs');
+const childProcess = require('child_process');
+
+class WindowsActions {
+    constructor(app) {
+        this.app = app;
+    }
+
+    async getCurrentSize() {
+        return this.app.browserWindow.getSize();
+    }
+
+    async setSize(width, height) {
+        await this.app.browserWindow.setSize(width, height);
+    }
+
+    async resizeWindows(width, height) {
+        await this.app.browserWindow.getBounds().then((bounds) => {
+            let x = bounds.x + (bounds.width - width);
+            let y = bounds.y + (bounds.height - height);
+            robot.setMouseDelay(500);
+            robot.moveMouse(bounds.x, bounds.y);
+            robot.mouseToggle("down");
+            robot.dragMouse(x, y);
+            robot.mouseToggle("up");
+        })
+    }
+
+    async getCurrentPosition() {
+        return this.app.browserWindow.getPosition();
+    }
+
+    async setPosition(x, y) {
```

Comment:
Remove x,y argument if you don't use them

Reply:
Updated